app/components/blacklight/facet_item_pivot_component.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Blacklight
+  # Render facet items and any subtree
+  class FacetItemPivotComponent < ::ViewComponent::Base
+    def self.id_counter
```

Comment:
Can you explain this counter? Does this just go up without bounds? Why do we need it?

Reply:
ARIA requires a unique ID to match the control with the thing it controls. I'm open to better ideas (blacklight-hierarchy uses `SecureRandom.uuid`, which is slow and.. unnecessary). 

Even at 1,000,000,000,000 cycles, the base 36 string is just 7 characters long, but I think most production servers will recycle threads well before that.

Reply:
That's super helpful context. Can you please add this information to a comment in this class?

Reply:
Sure; and I've gone ahead and capped the counter at ~1 million, and made sure we only add the id  to leaf nodes. If you have 1 million leaf nodes on  a  page, you might have a problem, but it probably isn't markup validity, accessibility or wonky javascript behavior.